roles/kubernetes/master/templates/kubeadm-config.v1alpha2.yaml.j2

```diff
@@ -1,7 +1,7 @@
 apiVersion: kubeadm.k8s.io/v1alpha2
 kind: MasterConfiguration
 api:
-{% if groups['kube-master'] | length > 1 and kubeadm_config_api_fqdn is defined %}
+{% if groups['kube-master'] | length >= 1 and kubeadm_config_api_fqdn is defined %}
```

Comment:
`groups['kube-master']` will always contain _at least_ `1` node. So doing a check for >= 1 is not needed. Then you can just remove that check.

Reply:
Done.